src/EditorFeatures/CSharp/DecompiledSource/CSharpDecompiledSourceService.cs

```diff
@@ -103,6 +106,11 @@ private Document PerformDecompilation(Document document, string fullName, Compil
 
             // Try to decompile; if an exception is thrown the caller will handle it
             var text = decompiler.DecompileTypeAsString(fullTypeName);
+
+            text += "#if false // Decompilation log" + Environment.NewLine;
```

Comment:
❗️  There a bunch of user-visible strings added in this pull request that need to be localized

Reply:
Where do I put the English versions?

Reply:
https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/CSharp/CSharpEditorResources.resx

Reply:
I've added the resource strings. I tried to follow the existing convention as closely as possible, if there's something wrong, please tell me. Thanks!